### ISSUE TITLE
Saving Global SEO settings overwrites customisations made to Google Tag Manager script templates

### ISSUE THREAD

**Issue by jamesmacwhite** [nystudio107/craft-seomatic#377]
I've made customisations to the head and body Google Tag Manager templates through the Tracking Scripts section. However, I've noticed when saving any Global SEO settings, the changes to these templates are wiped completely.

Saving the script template changes on the Tracking Scripts section, I can confirm they get committed to the database and are stored as JSON in the `metaContainers` column, but as soon as any global SEO settings are saved, the customisations are lost and get reverted back to the default templates.

**Steps to reproduce:**

1. Go to "SEOmatic CP Tracking Scripts". Enable the Google Tag Manager option and modify the default script templates for head and body. Save the changes and confirm the updated values are stored in the database on the `metaContainers` column
2.  Go to "SEOmatic CP Global SEO" and click save. You don't need to make any changes.
3. Return to the Tracking Scripts section and observe that previous customisations have been lost.

I've uninstalled and installed SEOmatic as fresh to make sure it's not something related to coming from Craft 2, but it does seem like a reproducible bug and not environment specific.

**Comment by jamesmacwhite** [comment]
Also occurs when saving Site Settings as well.

**Comment by khalwat** [comment]
Verified. Very strange...

**Comment by jamesmacwhite** [comment]
Thanks for verifying! I was getting slightly worried something was horribly wrong with the project I found this on. Not sure if other templates are affected like humans.txt or robots.txt.

**Comment by khalwat** [comment]
They aren't. I did a fancy special-casing for the scripts so that I could update things post-facto but it turns out to be a bad idea.

**Comment by jamesmacwhite** [comment]
I can understand why you'd do that. Unfortunately in my case, because I use GTM environments I've modified the original templates for supporting the different env params that get passed with the GTM container URL for dev/staging. I don't think there's any way to stop the override currently (other than not saving Global SEO/site settings!). I did think that maybe redirecting the template path from the original `__frontend/scripts/` paths, might work, but I think the whole settings in the metaContainers column just get reset, so it wouldn't help.
